src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -357,4 +361,45 @@ public void updateStage(UUID caseUUID, UUID stageUUID, Map<String, String> value
 
         camundaClient.completeTask(stageUUID, values);
     }
+
+    public ResponseEntity closeCase(UUID caseUUID) throws JsonProcessingException {
+        GetCaseworkCaseDataResponse caseDetails = caseworkClient.getCase(caseUUID);
+
+        //Get stage uuid from casework
+        String stage = caseworkClient.getActiveStage(caseDetails.getReference().replace("/", "%2F"));
```

Comment:
Unsure what the casework endpoint returns but mapping to a DTO would be preferred? I suspect there should only be 1 active stage for a case?

Reply:
its a Set<> but there should only be one - the set was for parallel stages.

Reply:
I've swapped this to a DTO, will make it more useful in the future if people want to get more out of it